### ISSUE TITLE
Usability features as enhancement

### ISSUE THREAD

**Issue by Sn1kk3r5** [JohanDegraeve/xdripswift#208]
When Libre algorythm is used the "calibration" Button on the Homescreen could be invisible.

When having a LIbre Sensor connected via direct BT (no Transmitter) the "Battery Alarm" could be invisible as well.

**Comment by Sn1kk3r5** [comment]
Multi Point calibration could be invisible as well when Libre Algorithm  is used

**Comment by liroyvh** [comment]
Why would you hide Multi-Point calibration option altogether? You need to see the toggle in order to know that its an option. :) Otherwise it becomes a bit confusing imho. However, what might be confusing is that its impossible to enable MPC when Libre Algorithm is selected. It might be unclear why. Maybe give alert that LA has to be disabled first?

And if you don't mind me adding a couple of suggestions:
1.) Display remaining sensor lifetime or show current lifetime (in days + hours, rather than only start time) in the "sensor" section. Right now it shows the start date & time, but not remaining lifetime. (Also: give alert when 14 days reached and only 12 hours left?)
2.) When using Libre2 direct: cease uploading battery status to Nightscout, as it will *always* upload 92% for some reason. Either don't upload anything or upload the device (iPhone) battery status instead. :)

**Comment by Sn1kk3r5** [comment]
Well my point was to have the main screen as empty as possible. Why would you need a feature when you can’t use it? 

Regarding a warning, not quite sure as I have the Libre App in parallel bit when I’m not mistaken there is a warning about sensor life span. 

Having the remaining time is a nice feature!

**Comment by liroyvh** [comment]
@Sn1kk3r5 Oh you mean just hiding "calibration" on homescreen? When you mentioned multi-point: this is only referenced in the CGM screen under Bluetooth, so I thought you meant hiding that as well when LA is enabled. But since that is the default: if you hide it, people won't know where it is or even know of its existence hehe. :P Hence my reservation if hiding that toggle would be a good idea.

But if you just mean homescreen: agreed, it's a pointless button when it does nothing. We can have free real-estate! ;)

**Comment by Sn1kk3r5** [comment]
You're right my point was a little misleading.

What I meant was:

Hiding the Calibration Button on the Home Screen when Libre Algorithm is used.

And.

Hiding Multipoint Calibration on the  connection setting screen.

This could be realized like the Nightscout Option in the settings screen in opposite direction.
When LA is activated make MPC invisible. Means at the beginning you see all options, after enabling LA just hide MPC.

Cause currently you can click on MPC and nothing is happening, no action and no information.

So in summary you have only the features available which are usable.
